fixture/test_thunderservice_mongohostdata_08.json

```diff
@@ -0,0 +1,131 @@
+{
```

Comment:
Is this file needed?

Reply:
Would you mind to delete this file? @anbrenna it's not needed

Reply:
done